src/plugins/dashboard/public/dashboard_container/embeddable/api/lib/extract_title_and_count.ts

```diff
@@ -15,5 +15,5 @@ export const extractTitleAndCount = (title: string): [string, number] => {
       return [baseTitle, Number(count)];
     }
   }
-  return [title, 1];
+  return [title, 0];
```

Comment:
Return zero instead of 1 when passed string is not of duplicate format. So we can increment whatever value returned from this function with reckless abandon on duplicate attempt, other wise if we choose not to do so, the first duplicate dashboard value would be of the form `dashboard name (2)`.